config/restrictions.yaml

```diff
@@ -364,4 +364,15 @@ restrictions:
     - "^python"
     - "^python-base"
     - "^ruby"
+  - path: "etcd-io/org.yaml"
+    allowedRepos:
+    - "^bbolt$"
+    - "^dbtester$"
+    - "^discovery.etcd.io$"
+    - "^discoveryserver$"
+    - "^etcd$"
+    - "^gofail$"
+    - "^jetcd$"
+    - "^raft$"
+    - "^website$"
```

Comment:
This list appears to be missing the following three repositories which we recently added `OWNERS` files to.

- [ ] `etcd-io/protodoc` 
- [ ] `etcd-io/etcdlabs` 
- [ ] `etcd-io/govanityurls`